client/components/hosted.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import HostedList from './hostedList.jsx';
+import axios from 'axios';
+import './hosted.css';
+
+class HostedBy extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
@pauly-van why are the objects in a single array? maybe more clear to make the state the object itself and then reference the individual properties